### PR TITLE
Default Shop Logistic Unit get from LogisticUnitGeneralSetting

### PR DESCRIPTION
**Pull request by likecodingloveproblems**
default post logistic units metrics were hard coded, Now they are generated from LogisticUnitGeneralSettings.

### CODE REVIEW

File: logistic/managers.py [hunk 1]
```diff
@@ -43,24 +43,50 @@ def generate_shop_logistic_units(self, shop):
...
+            models.LogisticUnitGeneralSetting.DoesNotExist, 
+            models.LogisticUnitGeneralSetting.MultipleObjectsReturned
+            ):
+            '''If we don't have post pishtaz and sefareshi in LogisticUnitGeneralSetting,
+            then we don't have to create post pishtaz and sefareshi in ShopLogisticUnit 
+            as default shop logistic units, but logistic team must be aware of this!'''
+
+    def get_default_post_metrics(self):
+        return {
+            'PPOST': models.LogisticUnitGeneralSetting.objects.get(
```
Comment (PouryaMansouri) [inline logistic/managers.py:83]: @likecodingloveproblems 
You got LogisticUnitGeneralSetting.DoesNotExist if the staff removes these instances.

I think it is better to have a method in this class that gets you the instance and if you don't have anyone create it for you.

I saw that one of the staff remove this instance from the admin panel.

Comment (PouryaMansouri) [reply logistic/managers.py:83]: @likecodingloveproblems

**Comment by mohsenone** [comment]
We can change LogisticUnitGeneralSetting model to contain all fields that should be created as a ShopLogisticUnit, and create our 5 default logistic unit objects in this table. Each time we want to create logistic units for a shop, we can use these 5 records as default units.

**Comment by likecodingloveproblems** [comment]
@mohsenone

**Review by movses-aslanian — APPROVED** [review]

**Pull request merged**